prime-router/docs/api/requeue.yml

```diff
@@ -11,7 +11,7 @@ paths:
     post:
       description: manually queue up a report to be sent again to one receiver
       security:
-        - OktaAuth: [ FUNCTION ]
+        - OAuth2: [ system_admin ]
       parameters:
```

Comment:
Is there a "least privilege" alternative?  There should be.

Reply:
`org_admin` might be the auth level to use here

Reply:
The actual permissions used/required needs review across all endpoints.

This should be address separately.